src/index.html

```diff
@@ -20,8 +20,692 @@
+        </div>
+      </header>
+      <main class="main">
+        <div
+          class="card"
+          data-qa="card"
+        >
+          <div class="card__img">
+            <img
+              src="images/imac.jpeg"
+              alt="image-Imac"
+            />
+          </div>
+
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina
+            <br />
+            5K Monoblock (MNED2UA/A)
+          </h2>
+          <p class="card__id">Product code: 195434</p>
+          <div class="card__review review">
+            <div class="review__stars">
+              <div class="stars stars--0">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <div class="stars stars--1">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <div class="stars stars--2">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <div class="stars stars--3">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <div class="stars stars--5">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+            </div>
+            <div class="review__points">Reviews: 5</div>
+          </div>
+          <div class="card__price-block">
+            <p class="card__price-title">Price:</p>
+            <p class="card__price-current">2,199</p>
+          </div>
+          <button
+            class="card__btn"
+            data-qa="card-hover"
+          >
+            buy
+          </button>
+        </div>
+        <div class="card">
+          <div class="card__img">
+            <img
+              src="images/imac.jpeg"
```

Comment:
Consider providing a more descriptive `alt` text for the product image to improve accessibility. Instead of 'image-Imac', use something like 'Apple iMac 27 Retina'.